chromadb/segment/impl/metadata/sqlite.py

```diff
@@ -564,7 +572,7 @@ def _value_criterion(
             col_exprs = [
                 table.string_value.isin(ParameterValue(_v))
                 if op == "$in"
-                else table.str_value.notin(ParameterValue(_v))
+                else table.string_value.notin(ParameterValue(_v))
```

Comment:
Is this the only real change?

Reply:
Why did tests not catch this bug? Can we add defensive tests before merging please